Jenkinsfile_CNP

```diff
@@ -103,7 +104,8 @@ env.DEFAULT_COLLECTION_ASSERTION_MODE="UNORDERED"
 env.MAX_NUM_PARALLEL_THREADS=6
 env.BEFTA_S2S_CLIENT_ID = "ccd_gw"
 env.ROLE_ASSIGNMENT_API_GATEWAY_S2S_CLIENT_ID = "ccd_data"
-env.BEFTA_TEST_STUB_SERVICE_BASE_URL = "http://ccd-test-stubs-service-aat.service.core-compute-aat.internal"
+// TODO: remove TEMPORARY OVERRIDE FOR CR&D, reinstate:     env.BEFTA_TEST_STUB_SERVICE_BASE_URL = "http://ccd-test-stubs-service-aat.service.core-compute-aat.internal"
+env.BEFTA_TEST_STUB_SERVICE_BASE_URL = "http://ccd-test-stubs-service-pr-166.service.core-compute-preview.internal"
```

Comment:
These values should be restored prior to merger.

Reply:
Fixed in [620a75a](https://github.com/hmcts/ccd-data-store-api/pull/2058/commits/620a75a1ae2c193d4cb5bea24a73c94f8643bf03)